### DISCUSSION TITLE
Hide section or individual pages from search index.

### DISCUSSION THREAD

**Discussion by johnnymatthews** [thuliteio/doks#961] [Q&A] [answered]
Is it possible to hide a particular page from the search index? Failing that, can I hide an entire section from the search index?

I currently have `searchSectionsShow` and `searchSectionsIndex` both set to `ALL`. I'm assuming I could add in each individual section I want to index, apart from the one that I don't. I'll give that a go now.

**Comment by johnnymatthews** [comment, marked as answer]
Been playing around with this a bit. Simply creating an array of strings for sections I want to index under `searchSectionsIndex` works fine. Leaving `searchSectionsShow` keeps the searchbar in the topbar nav.

```
searchSectionsShow = "ALL"
searchSectionsIndex = ["build", "install", "support", "developers"]
```

**Reply by h-enk** [reply]
This is how to exclude individual pages from the index: https://github.com/h-enk/doks/pull/799#issuecomment-1168224086

**Reply by johnnymatthews** [reply]
Oh nice! This might be useful! Thanks for the link!
